### DISCUSSION TITLE
Changing contents of comments

### DISCUSSION THREAD

**Discussion by nkhar** [mozilla-mobile/fenix#28781] [General]
Hello newbie here,

If I make any mistakes in the discussion please feel free to correct me and I will make the appropriate changes.

If I find some comments in the source code that have incorrect or obsolete information what should I do. Create an issue, report a bug, create a pull request. For this example I found that in Bitmap extension file: **org.mozilla.fenix.ext.Bitmap.kt**, in the function : **fun Bitmap.scaleToBottomOfView(view: ImageView)** on line 42 is the following comment:                 **// Majority of this math borrowed from // https://medium.com/@tokudu/how-to-whitelist-strictmode-violations-on-android-based-on-stacktrace-eb0018e909aa**. 

This link goes to a medium post discusses StrictMode PenaltyDeath reflection hacks. I don't think the link is relevant here and some other link was intended. such as: https://medium.com/mobile-app-development-publication/android-image-startcrop-and-endcrop-scaling-18d7297f1e7c

The StrictMode link is also used in **org.mozilla.fenix.perf.StrictModeManager** in the comments of function: **private fun StrictMode.ThreadPolicy.Builder.penaltyDeathWithIgnores(): StrictMode.ThreadPolicy.Builder** on line 182.

Can you please tell me what steps should I take to correct the link and also shat should be done in the future for such comments?

**Comment by kbrosnan** [comment]
To communicate with the team sign up for [Mozilla's Matrix server](https://wiki.mozilla.org/Matrix) and join the `#geckoview` or `#fenix` channels

This repository is going to be archived soon. Work has moved to https://github.com/mozilla-mobile/firefox-android/
